### PR TITLE
summary for 2016-02-25 meeting

### PR DESCRIPTION
**Pull request by veqtrus**

### CODE REVIEW

File: _posts/en/meetings/2016-02-25-meeting.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+## Participants
+
+    petertodd         Peter Todd
+    gmaxwell          Gregory Maxwell
+    btcdrak           btcdrak
+    morcos            Alex Morcos
+    sipa              Pieter Wuille
+    CodeShark         Eric Lombrozo
+    jonasshnelli      Jonas Schnelli
```
Comment (jonasschnelli) [inline _posts/en/meetings/2016-02-25-meeting.md]: nittypo: jonasschnelli

**Comment by jonasschnelli** [comment]
Nice. Thanks!

**Comment by veqtrus** [comment]
Fixed typo.

**Comment by btcdrak** [comment]
Thanks again.

ACK

**Pull request merged**